### DISCUSSION TITLE
Update Script

### DISCUSSION THREAD

**Discussion by APT256** [whatotter/pwnhyve#16] [Ideas]
For future update, would it be possible to do so either via a script or via apt instead of cloning the repo and installing over-the top?

**Comment by whatotter** [comment]
yeah that seems pretty smart, i could probably work on it regardless of having the screen - which somehow still hasn't arrived yet

**Reply by APT256** [reply]
If you ordered it via Amazon, I have been having issues with them lately as well. Especially if they are using USPS for shipping.
